spec/search_builders/hyrax/collection_search_builder_spec.rb

```diff
@@ -6,7 +6,7 @@
   let(:scope) { FakeSearchBuilderScope.new(current_user: user) }
 
   describe '#sort_field' do
-    its(:sort_field) { is_expected.to eq('title_si') }
+    its(:sort_field) { is_expected.to eq('title_ssi') }
```

Comment:
Including this fix because I believe it's a typo.

    title_si: The suffix _si suggests that this field should be treated as a "Short Integer" based on Solr's default dynamic field definitions.~
    title_ssi: The suffix _ssi suggests that this field is a "String, stored, and indexed".